src/modules/Plans/views/details/components/DetailsSection/components/StatusDetailsItem.tsx

```diff
@@ -1,18 +1,18 @@
 import type { FC } from 'react';
-import { PlanStatusCell } from 'src/modules/Plans/views/list/components/PlanStatusCell';
 import { DetailsItem } from 'src/modules/Providers/utils/components/DetailsPage/DetailItem';
 import { useForkliftTranslation } from 'src/utils/i18n';
+import PlanStatus from 'src/views/plans/list/components/PlanRowFields/PlanStatus/PlanStatus';
 
 import type { PlanDetailsItemProps } from './PlanDetailsItemProps';
 
-export const StatusDetailsItem: FC<PlanDetailsItemProps> = ({ resource }) => {
+export const StatusDetailsItem: FC<PlanDetailsItemProps> = ({ resource: plan }) => {
   const { t } = useForkliftTranslation();
 
   return (
     <DetailsItem
       title={t('Status')}
       helpContent={t('Migration plan state information and progress')}
-      content={<PlanStatusCell data={{ plan: resource }} fieldId={''} fields={[]} />}
```

Comment:
The spacing looks a little off

<img width="279" alt="Screenshot 2025-04-10 at 1 11 43 PM" src="https://github.com/user-attachments/assets/96336653-4d6b-48ac-84a3-4c6760239afd" />

Reply:
I don't really see any difference from main branch, and it seems like a minor bug, let's open a ticket for this and handle this outside of this PR.
screenshot from main:
![plan-status-spacing](https://github.com/user-attachments/assets/c8cb534e-394f-4e94-98d7-1bde2c8eeec4)